cypress.json

```diff
@@ -8,7 +8,8 @@
     "validAdminCode": "super",
     "validAdminPassword": "adminpassword",
     "fakeWard": "FAKENHS",
-    "baseUrl": "http://localhost:3001"
+    "baseUrl": "http://localhost:3001",
+    "ValidDupeWard": "super"
```

Comment:
is this cypress env var being used anywhere?

Reply:
no, its been removed